src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -479,3 +481,20 @@ def extrude_edges_up_to(
         else:
             self._grpc_client.log.info("Failed to extrude edges.")
             return []
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def rename_object(self, selection: List[EntityIdentifier], name: str) -> bool:
+        """Rename an object.
+
+        selection : List[str]
+            Selection of the object to rename.
+        name : str
+            New name for the object.
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
```

Comment:
Make sure the docstring is properly formatted. You need the "Parameters" title, the "Returns" section etc. Have a look at other methods